.mergify.yml

```diff
@@ -110,6 +110,20 @@ pull_request_rules:
         strict: false
       dismiss_reviews: {}
       delete_head_branch: {}
+  # release-1.6 branch
+  - name: automerge backport release-1.6
+    conditions:
+      - author=mergify[bot]
+      - base=release-1.6
+      - label!=do-not-merge
+      - 'status-success=DCO'
+      - 'status-success=continuous-integration/jenkins/pr-head'
```

Comment:
I don't think `continuous-integration` is relevant on release-1.6 anymore. I think we should call out the Github actions instead.